pkg/trace/sampler/scoresampler.go

```diff
@@ -7,6 +7,11 @@ package sampler
 
 import "github.com/DataDog/datadog-agent/pkg/trace/pb"
 
+const (
+	// If this is surpassed, the rate is set to 1.
```

Comment:
This is crystal clear and understandable, but for the sake of consistency, can we please use the correct Go style where the doc starts with the name of the identifier, just like we did here: https://github.com/DataDog/datadog-agent/blob/8700efc020d6468937c7831d5acbae92572bb8c4/pkg/trace/sampler/prioritysampler.go#L33-L34